matlab/test/test_ex_read_raw.m

```diff
@@ -26,3 +26,27 @@ function test_ex_read_raw_()
 assert(size(data, 2) == length(times))
 assertTrue(all(times < 53.01))
 assertTrue(all(times > 50.99))
+
+% import mne, numpy as np
+% data = np.random.RandomState(0).randn(3, 2)
+% info = mne.create_info(['MEG1', 'MEG2 with a very long name', 'STI 014'], 1000., ['grad', 'grad', 'stim'])
+% mne.io.RawArray(data, info).save('matlab/test/data/test_long_raw.fif')
+want_data = [[1.76405235, 0.40015721]; [0.97873798, 2.2408932]; [1.86755799, -0.97727788]];
+want_times = [0., 0.001]
+want_names = {'MEG1', 'MEG2 with a very long name', 'STI 014'};
```

Comment:
@robertoostenveld this test file was generated using the MNE-Python code above:

https://github.com/mne-tools/mne-matlab/pull/21/files#diff-b92edbcdf7fb511560a73a838a821de52f65f958b0910e05cafaff6c75808898

Reply:
the link in the comment seems to be self-referring to this comment itself. Where would I find the file?

Reply:
oh, sorry: I found it!